pkg/apis/submariner.io/v1/types.go

```diff
@@ -86,11 +86,11 @@ type GatewayStatus struct {
 // LatencySpec describes the round trip time information in miliseconds for a packet
```

Comment:
To preserve the granularity, shouldn’t the values use a finer unit, say microseconds or nanoseconds?

Reply:
Ok the time period gives it in nano seconds, I will change it to nano seconds.